RIS.js

```diff
@@ -1764,8 +1770,18 @@ function completeItem(item) {
 		if (cleanDOI) item.DOI = cleanDOI;
 	}
 
-	// hack for sites like Nature, which only use JA, journal abbreviation
-	if (item.journalAbbreviation && !item.publicationTitle) {
+	// Use the shorter journal abbreviation, and move the longer one to
```

Comment:
You're not worried that the logic here is getting a bit convoluted? It obviously works for the issue at hand, but it's hard to keep track of what's going on in the code, which will make future troubleshooting harder.
Otherwise this looks fine.